site/includes/sign-on-off.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+	<div class="row">
+{{#is signed "off"}}
```

Comment:
Twitter and Google are single sign-on scenarios, where the user is globally signed in to the entire system. The GC ecosystem does not have that, you can only sign in to specific apps - and in the vast majority if not all scenarios, it is unfortunately impossible to carry over your signed in state from one app to another. So it's actually misleading to include the login/out controls up in the global header.